os/drivers/compression/README.md

```diff
@@ -0,0 +1,62 @@
+# Compression driver
+
+This driver APIs are used to access compression and decompression functions of compression module.
+The ioctl function of the compress driver supports following commands and accordingly perform 
+following tasks .
+
+## 1. COMPIOC_COMPRESS cmd :
+
+For this cmd, user needs to pass a struct compress_header pointer as arg to the ioctl and need to store
+the members of the struct as below :
+input_size     ->  input size of the data <br />
+input_buffer   ->  input data <br />
+output_size    ->  should be same as input size because after compression, output size will be less than equal to input size <br />
+output_buffer  ->  should be allocated before passing to ioctl function 
+
+The ioctl function will call the compression function and compressed data will be stored in 
+output_buffer and output_size will get updated to the length of compressed data. It will usually be less than
+the passed output_size.
```

Comment:
Let us remove the last sentence "It will usually be less than the passed output_size."